### DISCUSSION TITLE
how do I fix this? sdvx for macbook

### DISCUSSION THREAD

**Discussion by bello89808m** [Drewol/unnamed-sdvx-clone#579] [Q&A]
How do I fix this? I followed every step before it but I'm currently stuck here because I don't know what the issue is and what I can do to fix it
<img width="585" alt="Screen Shot 2022-04-17 at 1 33 12 PM" src="https://user-images.githubusercontent.com/103875831/163732892-a4681375-61e3-4d5b-bee6-b5b69696187c.png">
.

**Comment by Hinara** [comment]
You might have forgot to execute `git submodule update --init --recursive ` at the root of the repository

**Reply by bello89808m** [reply]
what am I doing wrong here
<img width="534" alt="Screen Shot 2022-04-17 at 2 56 51 PM" src="https://user-images.githubusercontent.com/103875831/163733383-ee93bcbf-7299-4ee0-a849-9811ce9d9bef.png">

**Reply by RobinMEPITECH** [reply]
At the root of the repository means inside of the repository but at it's root, so you should enter into the repository first
Go into the repository first with `cd something/unnamed-sdvx-clone`
Then execute `git submodule update --init --recursive`

**Reply by bello89808m** [reply]
I got it working and its creating rn, but where should I find it after it's done?
<img width="682" alt="Screen Shot 2022-04-17 at 3 14 25 PM" src="https://user-images.githubusercontent.com/103875831/163734048-3041830b-6991-49c9-9690-15a9b80cf2e4.png">

**Reply by RobinMEPITECH** [reply]
That is just the configuration step, after that execute `cmake --build .`
If everything works fine the executable should be in the bin directory after that

**Reply by bello89808m** [reply]
is this normal or did I mess up?
<img width="682" alt="Screen Shot 2022-04-17 at 3 37 57 PM" src="https://user-images.githubusercontent.com/103875831/163734728-d67079df-0ebe-4a8f-b8e2-b2a4924d1c38.png">

**Reply by parakeet-live** [reply]
yeah it's just building rn
